### DISCUSSION TITLE
启用 FullCone NAT6 会导致 DHCPv6-PD 分配 IPv6 地址出现通信问题吗？

### DISCUSSION THREAD

**Discussion by cherinyy** [immortalwrt/immortalwrt#997] [Q&A] [answered]
我注意到在`网络`——`防火墙`中有个`启用 FullCone NAT6`的选项，但我是路由器 PPPoE 拨号，然后通过 DHCPv6-PD 分配 IPv6 地址。

所以我不需要启用 FullCone NAT6，但是我想知道如果启用了会导致出现通信问题吗？

**Comment by 1715173329** [comment, marked as answer]
不需要。你没配置 nat66 这个选项开了也无用。
